src/Core/Response/ResponseInterface.php

```diff
@@ -15,6 +15,6 @@ public function getRawBody(): string;
     /**
      * @return mixed
      */
-    public function getBody();
+    public function getBody(): mixed;
```

Comment:
@Floeig we are using PHP 7.2 as the minimum supported version this change would make the code uncompilable for developers with PHP 7.2 installed.

However, you can remove /** * @return mixed */ comment to get the desired change and remove Deprecation warning